### ISSUE TITLE
County results: Show township-level instead of county-level for New England states

### ISSUE THREAD

**Issue by alykat** [nprapps/elections24-general#8]
**Exploratory**
- [x] #146
- [x] Verify that AP is serving up EEVP for townships
- [x] #150

**Implementation**
- [x] #149
- [x] #148
- [x] #147
- [x] County results tables: Decide how to handle county-level unemployment data for New England townships
- [ ] https://github.com/nprapps/elections24-general/issues/198

-------

New England states:
* Connecticut
* Maine
* Massachusetts
* New Hampshire
* Rhode Island
* Vermont

Related new AP API attribute that might be useful:

> Township FIPS Code
> For New England states, the new `townFIPSCode` field in JSON indicates the township FIPS code of a reporting unit.

----

**Background**

Ticket ported over from the primaries rig:

Cities/towns may be more meaningful, and the maps are more granular. Noted in the [API docs](https://developer.ap.org/ap-elections-api/docs/index.html#t=Elections_Response_Examples.htm&rhsearch=county&rhhlterm=county&rhsyns=%20) — pulling RU rather than FIPS might get us there?

> Results by Reporting Unit
> This example (in XML and JSON) shows results by reporting unit for the primary races for Governor in Vermont. Since the level=ru parameter is specified in the request, the response includes results for both the specified state (Vermont) and each of the reporting units within this state (cities and towns in this example since Vermont is a New England state). Note that in addition to the reporting unit ID, the county FIPS code is also reported for each New England city or town, so that you can aggregate results to the county level.

> Results by FIPS Code
> This example (in XML and JSON) shows results by FIPS code for the primary races for Governor in Vermont. Since the level=fipscode parameter is specified in the request, the response includes results for both the specified state (Vermont) and each of the county FIPS codes (city/town results aggregated to the county level in this example since Vermont is a New England state).

Realistically, this is more a thought for the general than the primaries, but putting it here for now.

[AP documentation](https://developer.ap.org/ap-elections-api/docs/index.html#t=CT_FIPS_Codes_forPlanningRegions.htm&rhsearch=new%20england%20states&rhsyns=%20&rhhlterm=new%20england%20states) on mapping townships between the old counties and the new planning regions

Township-to-county [crosswalk documentation](https://www.census.gov/programs-surveys/geography/technical-documentation/county-changes/2020.html) from the Census Bureau

**Comment by alykat** [comment]
**About the township maps:**

generated maps all live in:
src/assets/synced/towns/STATE_POSTAL.svg

individual town paths have an ID that follows the pattern "fips-STATEFIPS-COUNTYFIPS-COUNTYSUB" (so `id="fips-09-140-04930"`) to match the pattern used by the township census data. we can update the script to use a different identifier later.

**Comment by alykat** [comment]
**Township identifiers**
I have attempted to match up the Census Bureau's identifiers with AP's in this spreadsheet:
https://docs.google.com/spreadsheets/d/17ZGP59tj7DTjBVTodSsKZv8f29ZqMFcgw9NQYRajXrI/edit?gid=0#gid=0

Maine is tricky: There are a few AP reporting areas that do not match up with Census, and a number of Census areas that are not reflected in AP. For the most part, everything else seems to match up.

**Township unemployment data**
I have aligned the Aug. 2024 township unemployment data with the state-county-countysub identifier we're using elsewhere. There are a handful of jurisdictions for which I have not been able to assign an identifier; the majority are fine.
